### ISSUE TITLE
Orbital Transfer Failing

### ISSUE THREAD

**Issue by c1derly** [UmbraSpaceIndustries/MKS#1466]
I'm trying to create an orbital transfer to another logistics module on the same planet and I keep getting the same error over and over.   "The target logistics module could not be found on the destination vessel".  I looked through the code and it looks like it can't find the orbital logistics module when it goes to deliver the goods.  I can't seem to get the source to build for some reason so I can't try to debug it to see what is going on.

**Comment by tjdeckard** [comment]
When a transfer is setup between two vessels, both the vessel ID and the logistics module part ID of the destination vessel are recorded. A check is performed right before delivery to make sure the destination logistics module is still attached to the vessel it was attached to when the transfer was initiated.

**Comment by c1derly** [comment]
Yes I read through the code and could tell that but the part is still attached to the base but it still unable to find it and fails.

**Comment by c1derly** [comment]
I seem to have found the root cause.  I got the orbital transfer working at least.  I started a new game and tried again and i could send transfers to one station but not the other way around.  I looked at the save file and realized that both modules on the different ships had the same module ID.  I changed it in the save file and was able to send orbital logistics.  This was from a saved ship that i launched 2 times.  I'm guessing the guid only gets generated when it first gets added to the ship and not every time the ship gets launched causing duplicate IDS to get generated.
